l10n_it_ricevute_bancarie/models/account.py

```diff
@@ -121,6 +121,30 @@ def month_check(self, invoice_date_due, all_date_due):
                 return True
         return False
 
+    def _post(self, soft=True):
+        inv_riba_no_bank = self.filtered(
+            lambda x: x.is_riba_payment
+            and x.move_type == "out_invoice"
+            and not x.riba_partner_bank_id
+        )
+        if inv_riba_no_bank:
+            inv_names = (
+                _(
+                    'Customer "%(customer_name)s", total %(amount)s',
+                    customer_name=inv.partner_id.display_name,
+                    amount=inv.amount_total,
+                )
+                for inv in inv_riba_no_bank
+            )
```

Comment:
Dentro `inv_names` mi sarei aspettato il `name`, o il `display_name`, o qualcosa per identificare la fattura `inv`, come mai non c'è? Potrebbe essere utile per capire in quali fatture compilare il conto mancante.

Reply:
Le fatture a quel punto non hanno ancora il nome perché non sono ancora state confermate. Né il `name` né il `display_name` mi sembravano particolarmente significativi al fine di riconoscere le fatture problematiche.

Piuttosto sarei per cambiare il nome alla variabile `inv_names` se può causare confusione.

Reply:
Il `display_name` è abbastanza furbo da usare `ref` se è valorizzato:
https://github.com/odoo/odoo/blob/f33e80dc4a98d0234d531d5f6a3950de124510bc/addons/account/models/account_move.py#L2382

quello dovrebbe permettere di identificare anche le fatture in bozza.

Se non usiamo `[display_]name` o simili il nome della variabile potrebbe essere `inv_details`.